src/main/java/teammates/ui/webapi/SearchStudentsAction.java

```diff
@@ -58,6 +98,10 @@ public JsonResult execute() {
                         s.getGoogleId()
                 );
             }
+            // If the course has been migrated, then the student would have been added already
+            if (isCourseMigrated(studentData.getCourseId())) {
```

Comment:
point for discussion: wondering if this is necessary @weiquu, I think other search actions use this as well, but I'm personally not a fan of this as `isCourseMigrated` incurs a DB transaction everytime, perhaps there's a better way to prevent duplicates from being entered (checking for equality of StudentData?)

Reply:
@yuanxi1 don't worry about this for now, we can merge this PR and make changes based on our discussion above in the near future

Reply:
@cedricongjh sure, that makes sense! Was my original approach as well, but I thought this looked neater (: anw will raise a pr to change it to equality check after all the search actions have been merged